frontend/src/components/Banner.js

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2021 Christoph Ehm <christoph.ehmendoerfer@campus.tu-berlin.de>
+
+import React from "react";
+import { useMediaQuery } from "react-responsive";
+
+import HeaderTypography from "./HeaderTypography";
+import ParagraphTypography from "./ParagraphTypography";
+import { Link } from "react-router-dom";
+import RoundedEdgesButton from "./RoundedEdgesButton";
+
+import { textColor, activeTextColor } from "../assets/jss/colorPalette";
+import bannerImage from "../assets/img/banner.png";
+
+/**
+ * Displays a welcoming background image with a button.
+ * @param user user object of the currently logged in account
+ * @returns {JSX.Element}
+ */
+export default function WelcomeBanner({ user }) {
+	const welcomeTitle = "Welcome to NFT The World!";
+	const welcomeSubtitle = <>
+		Breathtaking NFT shopping lounge.<br/>
+		Own your virtual map area from Nuremburg and Riga!
+	</>;
+	return <Banner user={user} title={welcomeTitle} subtitle={ welcomeSubtitle }/>;
+}
+
+/**
+ * Displays background banner with headline "title", text "subtitle" and login-dependent button.
+ * @param user user object of the currently logged in account
+ * @returns {JSX.Element}
+ */
+export function Banner({ subtitle, title, user }) {
+	const render=() => (
+		<div style={containerStyle}>
+			<div style={backgroundImageStyle}>
+				<div style={{ ...bannerItemStyle, marginBottom: "11px" }}>{ bannerTitle }</div>
+				<div style={{ ...bannerItemStyle, height: "66px", marginBottom: "32px" }}>{ bannerSubtitle }</div>
+				<div style={{ ...bannerItemStyle, }}>{ button }</div>
+			</div>
+		</div>
+	);
+
+	const largeSizeThreshold = 900;
```

Comment:
I would change this to 600, as between 600-900 nothing breaks and it still looks good.

Reply:
Okay, I found the sweet spot which is 700px. When I tested it in Firefox with 600px, it will reach temporarily 3 subtitle lines. 700px closely avoids that.